README-ko_kr.md

```
@@ -264,6 +264,22 @@ YouTube 플레이어 렌더링 방식을 제어하고 호환성을 위한 소스
 />
 ```
 
+**커스텀 플레이어 페이지**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)